### DISCUSSION TITLE
File Menu - Recent Documents - how to clear

### DISCUSSION THREAD

**Discussion by gazmanic** [DaxStudio/DaxStudio#788] [Q&A] [answered]
Hi,   How to reset/clear the Recent Documents list under File Menu?   Cheers Gary

**Comment by dgosbell** [comment, marked as answer]
There is no way to do this using the UI at the moment (it's one of the low priority items on my list)

But the list is stored in the registry at `Computer\HKEY_CURRENT_USER\SOFTWARE\DaxStudio\FileMRU` so you could edit / clear out the items there if you are confident editing the registry. If you delete the whole `FileMRU` item it will clear the whole list and DAX Studio will recreate that key when you next save a file.
